iothub/device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/ClientConfiguration.java

```diff
@@ -41,6 +41,8 @@
     public static final int DEFAULT_AMQP_OPEN_AUTHENTICATION_SESSION_TIMEOUT_IN_SECONDS = 20;
     public static final int DEFAULT_AMQP_OPEN_DEVICE_SESSIONS_TIMEOUT_IN_SECONDS = 60;
 
+    public static final int DEFAULT_SEND_INTERVAL_IN_MILLISECONDS = 10;
```

Comment:
Shouldn't the default value here be 50, so that it isn't a breaking change?

Reply:
The default send interval at the top level ClientOptions was 10 (and remains 10), so I consider this a bug fix to go from 50 to 10 here in the AMQP layer

Reply:
So, the 50 was more of an artificial delay than advertised feature behavior? 👍🏽

Reply:
Exactly